### PR TITLE
chore: improve handbook comms checklists

### PR DESCRIPTION
**Pull request by krol3**
#### What this PR does / why we need it:
Improve handbook communications team

#### Which issue(s) this PR fixes:
Related to https://github.com/kubernetes/sig-release/issues/2341

#### Special notes for your reviewer:
cc: @reylejano @katcosgrove  @fsmunoz  @Priyankasaggu11929 @sftim 
cc: @kubernetes/sig-docs-blog-owners

### CODE REVIEW

File: release-team/role-handbooks/communications/README.md [hunk 1]
```diff
@@ -122,7 +141,7 @@ This is an example of a typical release cycle and the order of how tasks will fl
         <td>
         <ul>
         <li>Start attending the Release Team weekly meeting
-        <li>Join <code>#sig-release</code> and <code>#release-comms</code> Slack channels
+        <li> Join <code>#sig-release</code> and <code>#release-comms</code> Slack channels
```
Comment (kcmartin) [inline release-team/role-handbooks/communications/README.md:144]:
```suggestion
        <li> Join the following Slack channels: <code>#sig-release</code>, <code>#release-comms</code>, <code>#sig-docs</code>, and <code>#sig-docs-blog</code>.
```

**Review by kcmartin — CHANGES_REQUESTED** [review]

File: release-team/role-handbooks/communications/README.md [hunk 2]
```diff
@@ -22,13 +22,16 @@ Communications deliverables that come from the release process include a release
...
+The Kubernetes release cycle spans 15 weeks, however it may run longer. The typical workload for the communications team is very light during the first few weeks of the release cycle. In the later weeks, **the workload can become heavy, and will continue a few weeks after the release.**
 
 The expected time investment for both leads and shadows are as follows:
 
 - 30 minutes to 2 hours a day (depending upon week), requesting and reviewing incoming KEPs and blog PRs, working with other SIGs or the CNCF to manage the feature blog posts, and following Slack channels in order to keep pending content current.
 
-- 1 to 5 hours a week, attending Release Team (weekly) and Burndown meetings (daily during Code Freeze)
+- 1 to 5 hours a week, attending
+    - Release Team (weekly) 
+    - SIG-Docs Team (biweekly) and
```
Comment (katcosgrove) [inline release-team/role-handbooks/communications/README.md:33]:
```suggestion
    - SIG Docs Team (biweekly) and 
```

File: release-team/role-handbooks/communications/README.md [hunk 3]
```diff
@@ -180,7 +199,7 @@ This is an example of a typical release cycle and the order of how tasks will fl
         <ul>
         <li>Work with Enhancements and Release Note leads to determine which deliverables are most noteworthy post-Enhancements Freeze
         <li>In the coming weeks, follow the progress of these enhancements, as they will roll into the Major Themes and be called out and described in the release blog
-        <li>Work with the Release Team and decide if the release warrants a mid-cycle 'Deprecations and Removals' blog
+        <li>Work with the Release Team and decide if the release warrants a mid-cycle 'Deprecations and Removals' blog. Generally, this is decided around a minimal of 2 deprecations and removals, or if there are significant deprecations and removals that will impact the community.
```
Comment (katcosgrove) [inline release-team/role-handbooks/communications/README.md:202]:
```suggestion
        <li>Work with the Release Team and decide if the release warrants a mid-cycle 'Deprecations and Removals' blog. Generally, this is decided at a minimum of two deprecations or removals, or if there are significant deprecations and removals that will impact the community. 
```

Comment (katcosgrove) [reply release-team/role-handbooks/communications/README.md:202]: Do we want to clarify this further to say that any deprecation or removal that has the potential to cause widespread confusion or panic regardless of its actual impact should justify a mid-cycle deprecations and removals blog? Fully okay with a no on that one, I tend to err way on the side of caution

File: release-team/role-handbooks/communications/README.md [hunk 4]
```diff
@@ -69,32 +72,48 @@ The release lead will drive the content for the release theme and logo.
...
+The suggested deadlines to be supported by the current Comms team:
+- Deadline - PRs blog place holder: Week 09.
+- Deadline - PRs ready for review: Week 11
+
+> Please note that the release team will support the blog publication after the release day too. 
 
 ### Mid-cycle deprecations and removals blog
 
-This blog is optional and will vary from release to release. Work with the rest of the release team ahead of the Code Freeze date to determine if a mid-cycle blog focused on feature deprecations and removals is warranted. If so, facilitate its creation and publication. You can create a Slack thread on [#sig-release](https://kubernetes.slack.com/archives/C2C40FMNF) to discuss this.
+This blog is optional and will vary from release to release. Work with the rest of the release team ahead of **the Code Freeze date to determine if a mid-cycle blog focused on feature deprecations and removals is warranted**.
```
[the remark this answers is not in the document]
Comment (krol3) [reply release-team/role-handbooks/communications/README.md:99]: @a-mccarthy  could you add it as suggestion to commit it ?

File: release-team/role-handbooks/communications/release-blog.md [hunk 5]
```diff
@@ -72,6 +72,9 @@ slug: kubernetes-1-XX-release-announcement
 
 <INTRO>
 
+## Release theme and logo
+<Logo image size is recommended as maximum 2160px>
```
Comment (kcmartin) [inline release-team/role-handbooks/communications/release-blog.md:76]:
```suggestion
<Logo image size is recommended to be no more than 2160px>
```

File: release-team/role-handbooks/communications/README.md [hunk 6]
```diff
@@ -69,39 +72,84 @@ The release lead will drive the content for the release theme and logo.
 
 Tracking, facilitating, and organizing the publication of the Feature Blog series is a major deliverable of the Comms team. Feature blogs are opt-in for SIGs, and authored by enhancement developers and others close to the features. It helps to work with the release lead and use the respective SIG Slack channels to remind the SIGs about opting-in to feature blogs.
```
Comment (kcmartin) [inline release-team/role-handbooks/communications/README.md:73]:
```suggestion
Tracking, facilitating, and organizing the publication of the Feature Blog series is a major deliverable of the Comms team. Feature blogs are opt-in for SIGs, and are authored by enhancement developers and others close to the features. We do, however, need to encourage owners of important enhancements to opt in to writing feature blogs. Examples of important enhancements that warrant a feature blog might include: breaking changes, features that have been in progress for a long time and are graduating, and features that are considered mandatory by the Release Lead. It helps to work closely with the Release Lead and use the respective SIG Slack channels to remind the SIGs about opting in to feature blogs and provide any necessary context to blog authors. 
```

**Review by kcmartin — CHANGES_REQUESTED** [review]
Adding a section about encouraging selective feature blog participation

**Comment by sftim** [comment]
I added a thumbs-up reaction to https://github.com/kubernetes/sig-release/pull/2355#discussion_r1425675394 but I also want to call out how valuable it is to get writeups that match the features and changes our users really care about.

**Comment by kcmartin** [comment]
> I added a thumbs-up reaction to [#2355 (comment)](https://github.com/kubernetes/sig-release/pull/2355#discussion_r1425675394) but I also want to call out how valuable it is to get writeups that match the features and changes our users really care about.

Thanks @sftim -- I added an item to the list of examples to reflect your comment

File: release-team/role-handbooks/communications/README.md (hunk 6 above)
Comment (reylejano) [reply release-team/role-handbooks/communications/README.md:73]: Enhancements that will be opted-in as a major theme may be a good candidate for a feature blog
This also gets the enhancement owner or SIG to think if their enhancement should be included in the major themes of the release

Comment (krol3) [reply release-team/role-handbooks/communications/README.md:73]: @reylejano  could you included as suggestion in the handbook? 🙏🏾

File: release-team/role-handbooks/communications/README.md [hunk 7]
```diff
@@ -22,13 +22,16 @@ Communications deliverables that come from the release process include a release
...
 
 - 30 minutes to 2 hours a day (depending upon week), requesting and reviewing incoming KEPs and blog PRs, working with other SIGs or the CNCF to manage the feature blog posts, and following Slack channels in order to keep pending content current.
 
-- 1 to 5 hours a week, attending Release Team (weekly) and Burndown meetings (daily during Code Freeze)
+- 1 to 5 hours a week, attending
+    - Release Team (weekly) 
+    - SIG Docs Team (biweekly) and 
+    - Burndown meetings (daily during Code Freeze).
 
 **NOTE:** These are estimates and your personal experience may vary. The more time you spend working as a team and regularly communicating within your team, the better off your experience will be.
```
Comment (kcmartin) [inline release-team/role-handbooks/communications/README.md:36]:
```suggestion
**NOTE:** These are estimates and your personal experience may vary. The keys to success in this role are collaboration with the team and maintaining regular communication within the team.
```

File: release-team/role-handbooks/communications/README.md [hunk 8]
```diff
@@ -69,39 +72,84 @@ The release lead will drive the content for the release theme and logo.
...
+> Note that blog PRs in k/website are dated, and automation will publish future-dated entries. This enables a PR process decoupled from blog publication date.
 
-As feature blogs are opted in, assign them to shadows and yourself for tracking and facilitation. The responsibility is to ensure the blog authors have the resources and information they need, including editorial and tech reviews once ready.
+**As feature blogs are opted in**, assign them to shadows and yourself for tracking and facilitation. The responsibility is to ensure the blog authors have the resources and information they need, including editorial and tech reviews once ready.
 
-Work closely with the SIG Docs Blogs team (connecting on `#sig-docs-blogs` and by attending meetings), as they are typically available for editorial reviews. Share with them the feature blog schedule and updates throughout the cycle.
+**Work closely with the SIG Docs Blogs team** (connecting on `#sig-docs-blogs` and by attending meetings), as they are typically available for editorial reviews. Share with them the feature blog schedule and updates throughout the cycle.
```
Comment (kcmartin) [inline release-team/role-handbooks/communications/README.md:80]:
```suggestion
**Work closely with the SIG Docs Blogs team** (Communicate with them via `#sig-docs-blogs` and by attending meetings), as they are typically available for editorial reviews. Share the feature blog schedule and updates with them throughout the cycle.
```

File: release-team/role-handbooks/communications/README.md [hunk 9]
```diff
@@ -69,39 +72,84 @@ The release lead will drive the content for the release theme and logo.
...
+- [Deprecated APIs Removed in 1.16](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/)
+- [Kubernetes Removals and Major Changes In 1.25](https://kubernetes.io/blog/2022/08/04/upcoming-changes-in-kubernetes-1-25/)
 
-Publication should occur ahead of the release in order to inform the community and allow for preparation time. Start the discussion mid-cycle and well ahead of Code Freeze, and target publication for Code Freeze week.
+> Publication should occur ahead of the release in order to inform the community and allow for preparation time. Start the discussion mid-cycle and well ahead of Code Freeze, and target publication for Code Freeze week.
 
 ### Press and release webinar
 
-This is a light but very important component of the Communications Coordinator role. Two sets of activities need to be scheduled with the CNCF, namely press release and interview scheduling around the release day and the release webinar after the release.
+This is a light but very important component of the Communications Coordinator role. Two sets of activities need to be scheduled with the CNCF:
```
Comment (kcmartin) [inline release-team/role-handbooks/communications/README.md:114]:
```suggestion
This is a simple but very important component of the Communications Coordinator role. Two sets of activities need to be scheduled with the CNCF:
```

**Review by kcmartin — CHANGES_REQUESTED** [review]
Added  a few copy edits.

**Review by kcmartin — APPROVED** [review]
Thanks for approving my suggestions, the rest looks good!

**Pull request merged**